src/js/utils/browser.js

```diff
@@ -85,6 +85,14 @@ export const IE_VERSION = (function() {
 
 export const IS_SAFARI = (/Safari/i).test(USER_AGENT) && !IS_CHROME && !IS_ANDROID && !IS_EDGE;
 export const IS_ANY_SAFARI = IS_SAFARI || IS_IOS;
+export const DESKTOP_SAFARI_VERSION = (function() {
```

Comment:
Maybe this should just be `SAFARI_VERSION`? on iOS it'll be equivalent to `IOS_VERSION`?
I.e., if you return `IOS_VERSION` instead of `null` from this iife.

Reply:
Personally I would find it clearer to have `IOS_VERSION` solely responsible for mobile and `DESKTOP_SAFARI_VERSION` solely responsible for desktop, rather than a multi-purpose `SAFARI_VERSION`, but that's just me. Happy to make this change if you want me to.

Reply:
@videojs/core-committers any thoughts?

Reply:
I think having a separate Desktop Safari version would be more clear.

Reply:
Yeah I think having desktop is better as well